.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/source/"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 2
```

Comment:
Just curious why limit the open pull requests to 2. My understanding is, if Dependabot has raised 2 PRs, the latest one is going to contain the latest version bumps, so we'll just want to merge the latest PR and ignore the first. Perhaps just limit this to 1 is enough?

Reply:
I think (unvalidated) that Dependabot uses multiple PRs when there's major version bumps and dependencies between libraries, to decouple a bit of the noise and not just "move everything to latest" all in one go.

I chose 2 to limit the amount of stuff we need to clean up if it doesn't behave as we expect (this will be a bit of a discovery journey for me, I've never used Dependabot before), but also to see how it operates with multiple PRs at a time.

We can change this later if we need.